.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            max-parallel: 1
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                include:
+                    - description: 'Symfony 3.4'
+                      php: '7.2'
+                      composer_option: '--prefer-lowest'
+                    - description: 'Symfony 4.4'
+                      php: '7.3'
+                      symfony: 4.4.*
+                    - description: 'Symfony 5.1'
+                      php: '7.4'
+                      symfony: 5.1.*
+                    - description: 'Symfony 5.2'
+                      php: '8.0'
+                      symfony: 5.2.*
+        name: PHP ${{ matrix.php }} tests (${{ matrix.description }})
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Cache
+              uses: actions/cache@v2
+              with: 
+                  path: ~/.composer/cache/files
+                  key: ${{ matrix.php }}-${{ matrix.symfony }}-${{ matrix.composer_option }} 
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+            - run: |
+                  sed -ri 's/"symfony\/(.+)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
+              if: matrix.symfony
+            - run: composer update --no-interaction --no-progress --ansi ${{ matrix.composer_option }}
+            - run: vendor/bin/simple-phpunit
```

Comment:
the `composer validate --strict --no-check-lock` test got lost. We should have it on CI (maybe with a separate job as we don't need to do it for all PHP versions and we don't even need to install deps for that)